alcs-frontend/src/app/features/admin/ceo-criterion/ceo-criterion.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+  <div class="split">
+    <h3>CEO Criterion</h3>
+    <button mat-flat-button color="primary" (click)="onCreate()">Create</button>
+  </div>
+
+  <table mat-table [dataSource]="ceoCriterion">
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+
+    <ng-container matColumnDef="number">
+      <th mat-header-cell *matHeaderCellDef>Number</th>
+      <td mat-cell *matCellDef="let row">{{ row.number }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="label">
+      <th mat-header-cell *matHeaderCellDef>Label</th>
+      <td mat-cell *matCellDef="let row">{{ row.label }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="description">
+      <th mat-header-cell *matHeaderCellDef>Description</th>
+      <td mat-cell *matCellDef="let row">{{ row.description }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="code">
+      <th mat-header-cell *matHeaderCellDef>Code</th>
+      <td mat-cell *matCellDef="let row">{{ row.code }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="actions">
+      <th mat-header-cell *matHeaderCellDef>Actions</th>
+      <td mat-cell *matCellDef="let row">
+        <button *ngIf="row.code !== 'MODI'" class="edit-btn" mat-flat-button (click)="onEdit(row)">
```

Comment:
nothing wrong, just curious why Modi is an exception.

Reply:
MODI is the special citerion that has additional rules attached, so we do not want to let users edit it